Sources/SpeziOnboarding/OnboardingFlow/NavigationPath+Codable.swift

```diff
@@ -56,7 +56,7 @@ extension NavigationPath {
     /// Function that provides access to the last element on top of the `NavigationPath` that satisfies a certain predicate
     ///
     /// - Parameters:
-    ///   - where: The predicate determining if the element of the `NavigationPath` is considered
+    ///   - predicate: The predicate determining if the element of the `NavigationPath` is considered
```

Comment:
Thanks for the fixes, didn't know that one has to specify the internal parameter name (makes only limited sense from my point of view?)

Reply:
Honestly, this isn’t clearly documented nor communicated. Saw sources stating both works.
But generally, it seems that stating the parameter name instead of the argument label is preferred.
* It‘s always present (e.g. `_` label which is not unique)
* Parameter names are typically more descriptive as Argument labels sometimes try to „flow natural“ with the overall function signature.
* And it’s consistent with the rest of the Spezi codebase :)

You may refer to https://developer.apple.com/documentation/xcode/writing-symbol-documentation-in-your-source-files#Describe-the-parameters-of-a-method

Reply:
Interesting, thanks for the detailed explanation! :)